style-requirements.txt

```diff
@@ -2,6 +2,7 @@ cmake-format
 flake8
 flake8-bugbear
 flake8-simplify
+Flake8-pyproject
```

Comment:
lower-case 'f' to be consistent?

Reply:
(and maybe keep alphabetically sorting)

Reply:
Using upper-case because it's what the project is called: https://pypi.org/project/Flake8-pyproject/

I've changed the order to alphabetical.